### PR TITLE
Adds the files I will need to create the homepage of the website.

### PR DESCRIPTION
**Pull request by valvarez1218**
Adds the servlet file for retrieving data and the js file where I will create the organize the data, create HTML elements, etc. 

I am planning to use index.html as the homepage. Is this fine?

### CODE REVIEW

**Comment by carinegordillo** [comment]
I am currently using the index.html as the create event page before we try to make it a pop up, I can move it into a different file if you'd like though @valvarez1218

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -4,6 +4,10 @@
         <meta charset="UTF-8">
         <title>My Portfolio</title>
         <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" 
+            href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" 
+            integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" 
+            crossorigin="anonymous">
```
Comment (jmuzy-sps) [inline src/main/webapp/index.html:10]: I had never heard of these before, learned something new today!

**Review by jmuzy-sps — APPROVED** [review]
@carinegordillo (and maybe @valvarez1218 in this pull request?) could you rename index.html to something like createevent.html? Index usually refers to the homepage/main page so just want your file name to accurately reflect its contents

**Review by josh-hdz — APPROVED** [review]

**Comment by josh-hdz** [comment]
I added the basic imports and class for fetch servlet

**Pull request merged**